docs/actions-cohortextractor.md

```diff
@@ -60,18 +60,14 @@ generate_study_cohort
   outputs:
     highly_sensitive:
       data: output/input.csv.gz
-    ### This file is produced in compressed format. If using Stata for your analysis you may need to change
-    ### the --output-format option to 'csv' and the .gz file extension to produce an uncompressed CSV instead.
 ```
 
-This produces a zipped file to reduce storage space required in the backend. Changing to `--output-format=csv` argument allows an uncompressed CSV to be produced which may be useful while testing in dummy data or if using Stata for analysis of this file. To import the compressed file in Python, you can use `pd.read_csv(<file>, compression='gzip')`. 
+Running the action with `opensafely run generate_study_population` will create the file `output/input.csv.gz`. The file is compressed by default to reduce storage space required and improve performance in the backend.  If you need to view the csv locally for debugging your code, running `gunzip output/input.csv.gz` in a terminal (use git-bash in windows) should give you a plain `.csv` to view.
 
 The size of the dummy dataset is determined by the `population_size` option [in the `project.yaml`](actions-pipelines.md#project-yaml-format).
 
-Running the action with `opensafely run generate_study_population` will create the file `output/input.csv.gz`.
-
 Running the action again will overwrite the existing dataset with a new one (check the file's _modified_ date) which will contain different data than previously (even if the `study_definition.py` didn't change) because the values are randomly generated.
-Beware that on Windows, you can't have `input.csv(.gz)` open and generate a new one at the same time.
+Beware that on Windows, you can't have `input.csv.gz` open and generate a new one at the same time.
```

Comment:
suggestion: Not your content, but we could change this while we're here to stand out too:

```suggestion

!!! warning
    On Windows, you can't have `input.csv.gz` open and generate a new `input.csv.gz` at the same time.
```

Reply:
Hmm, I think this actually needs changing. Given that they will likely never have a input.csv.gz open anymore, as they will be looking at the uncompressed `input.csv`, maybe the whole thing should go?

Reply:
:arrow_up: I also thought that too. I don't know if other users might do something like "open in a GUI app" which might lock the file?

But feel free to remove it, if you don't think it's needed.

Reply:
I've removed it.